openapi/servicenow/client.bal

```diff
@@ -0,0 +1,477 @@
+    # + sysparmExcludeReferenceLink - Flag that indicates whether to exclude Table API links for reference fields
+    # + sysparmFields - Comma-separated list of fields to return in the response.
+    # + sysparmInputDisplayValue - Flag that indicates whether to set field values using the display value or the actual value.
+    # + sysparmQueryNoDomain - Flag that indicates whether to restrict the record search to only the domains for which the logged in user is configured.
+    # + sysparmView - UI view for which to render the data. Determines the fields returned in the response.
+    # + accept - Data format of the response body.
+    # + contentType - Data format of the request body.
+    # + xNoResponseBody - By default, responses include body content detailing the new record. Set this header to true in the request to suppress the response body.
+    # + return - If successful, returns a JSON object that includes updated record of the table. Otherwise returns the relevant error.
+    remote isolated function updateRecordById(string tableName, string sys_id, json payload, string? sysparmDisplayValue = (), boolean? sysparmExcludeReferenceLink = (), string? sysparmFields = (), boolean? sysparmInputDisplayValue = (), boolean? sysparmQueryNoDomain = (), string? sysparmView = (), string? accept = (), string? contentType = (), string? xNoResponseBody = ()) returns json|error {
+        string  path = string `/api/now/table/${tableName}/${sys_id}`;
+        map<anydata> queryParam = {"sysparm_display_value": sysparmDisplayValue, "sysparm_exclude_reference_link": sysparmExcludeReferenceLink, "sysparm_fields": sysparmFields, "sysparm_input_display_value": sysparmInputDisplayValue, "sysparm_query_no_domain": sysparmQueryNoDomain, "sysparm_view": sysparmView};
+        path = path + check getPathForQueryParam(queryParam);
+        map<any> headerValues = {"Accept": accept, "Content-Type": contentType, "X-no-response-body": xNoResponseBody};
+        map<string|string[]> accHeaders = getMapForHeaders(headerValues);
+        http:Request request = new;
+        request.setPayload(payload);
+        json response = check self.clientEp->put(path, request, headers = accHeaders, targetType=json);
+        return response;
+    }
+    # Deletes the specified record from the specified table.
+    #
+    # + tableName - Name of the table from which to delete the specified record, such as "incident" or "asset".
+    # + sys_id - Sys_id of the record to delete.
+    # + sysparmQueryNoDomain - Flag that indicates whether to restrict the record search to only the domains for which the logged in user is configured.
+    # + accept - Data format of the response body.
+    # + return - **HTTP Status Code:** `204`. Meeting updated.
+    remote isolated function deleteRecordById(string tableName, string sys_id, boolean? sysparmQueryNoDomain = (), string? accept = ()) returns http:Response|error {
```

Comment:
Here also function name must be `deleteRecord`.

Reply:
Fixed by https://github.com/kasthuriraajan/ballerinax-openapi-connectors/commit/7d722e1656dd4084768a6a6f0aab1753b97a5d98